modules/libcom/src/iocsh/libComRegister.c

```diff
@@ -64,7 +66,8 @@ IOCSH_STATIC_FUNC void echo(char* str)
 
 static const iocshArg echoArg0 = { "string",iocshArgString};
 static const iocshArg * const echoArgs[1] = {&echoArg0};
-static const iocshFuncDef echoFuncDef = {"echo",1,echoArgs};
+static const iocshFuncDef echoFuncDef = {"echo",1,echoArgs,
+                                         "Print any string (can aslo print EPICS environment variables)\n"};
```

Comment:
aslo -> also

Maybe: "Print string after expanding macros and environment variables".

Reply:
adjusted to suggestion in 8711dcb7b57d0627de06b3cae10ff73dc8a84095